### PR TITLE
feat(tools): no more global installs/exports

### PR DESCRIPTION
**Pull request by Bouncey**

### CODE REVIEW

**Comment by Bouncey** [comment]
I can't access the synk report. Any ideas? @raisedadead @ojongerius

**Review by raisedadead — APPROVED** [review]

**Comment by ojongerius** [comment]
@Bouncey neither can I, @raisedadead can you help us out here?

**Comment by raisedadead** [comment]
It should be public for everyone. I am checking the configuration once again.

**Comment by raisedadead** [comment]
This needs a rebase. Better yet doing an `npm install` should be a good idea and then amending the commit.

File: package.json [hunk 1]
```diff
@@ -36,12 +36,14 @@
...
     "aws-lambda": "^0.1.2",
     "bluebird": "^3.5.1",
+    "dotenv": "^5.0.1",
     "graphql": "0.13.2",
     "graphql-playground-middleware-lambda": "^1.5.1",
     "graphql-tools": "2.24.0",
     "merge-graphql-schemas": "1.5.1",
     "moment": "^2.20.1",
     "mongoose": "^5.0.12",
+    "serverless": "^1.26.1",
```
Comment (ojongerius) [inline package.json]: Why did you add this here? Edit: I like this, you are assuring we have a compatible version of the serverless framework.

File: serverless.yml [hunk 2]
```diff
@@ -8,8 +8,8 @@ provider:
   stage: dev
   region: us-east-1
   environment:
-    MONGODB_URL: ${env:MONGODB_URL}
-    GRAPHQL_ENDPOINT_URL: ${env:GRAPHQL_ENDPOINT_URL}
+    MONGODB_URL: ${file(./config/secrets.js):getSecret.MONGODB_URL}
```
Comment (ojongerius) [inline serverless.yml]: Shiny!

**Review by ojongerius — APPROVED** [review]
Nice one!

**Comment by Bouncey** [comment]
We need `serverless`, the vulnerability was there even when it was treated as a global.

**Comment by ojongerius** [comment]
These changes were smuggled in using #56, closing.
